### PR TITLE
small improvements to the documentation

### PR DESCRIPTION
**Pull request by areski**

### CODE REVIEW

**Comment by areski** [comment]
I'm very new to Pyramid and Cornice!
This is just new comer going through the doc for the first time, noting what he found strange and trying to improve some style in few places.

**Comment by ametaireau** [comment]
Great, it's very valuable to have newcomers feedback on this, especially because this part of the docs were written a long time ago.

File: docs/source/resources.rst [hunk 1]
```diff
@@ -8,7 +8,8 @@ registered as handlers for the appropriate methods / services.
 Here is how you can register a resource::
 
     from cornice.resource import resource, view
-    
+
+    USERS = {1: {'name': 'gawel'}, 2: {'name': 'tarek'}}
```
Comment (leplatrem) [inline docs/source/resources.rst]: missing underscore prefix ?

**Comment by leplatrem** [comment]
Impressive amount of relevant fixes :) thanks !

File: docs/source/resources.rst (hunk 1 above)
Comment (areski) [inline docs/source/resources.rst]: great catch :) fixed here https://github.com/mozilla-services/cornice/commit/9441b1936a8cff845707f182bb8d41494a2b831d

**Comment by leplatrem** [comment]
Tests fail with python3 in setup... thus unrelated to this PR AFAIK

**Comment by areski** [comment]
The tests break due to rxjson, PR at https://github.com/spiral-project/rxjson/pull/8

If you need to test you can replace the 3 "rxjson" in tox.ini by "git+https://github.com/areski/rxjson.git@e5c4e3293e66a9f8c7e10e31b1a1300975cb404a"

Let's wait this get merged to rxjson

**Comment by ametaireau** [comment]
This has been merged!

**Pull request merged**